test/gtest/uct/uct_test.h

```diff
@@ -186,6 +186,32 @@ class uct_test : public testing::TestWithParam<const resource*>,
         uct_iov_t               m_iov;
     };
 
+    class uct_tl_names {
+    public:
+        uct_tl_names() {
+            ib_ud_tl_names.push_back("ud");
+            ib_ud_tl_names.push_back("ud_mlx5");
+
+            ib_rc_tl_names.push_back("rc");
+            ib_rc_tl_names.push_back("rc_mlx5");
+
+            ib_rc_dc_tl_names.push_back("dc_mlx5");
+        }
+        std::vector<std::string> get_ib_ud() const {
+            return ib_ud_tl_names;
+        }
+        std::vector<std::string> get_ib_rc() const {
+            return ib_rc_tl_names;
+        }
+        std::vector<std::string> get_ib_rc_dc() const {
```

Comment:
return std::vector<std::string>& instead of copy of vector

Reply:
fixed